!SVG/vercel.svg

```diff
@@ -0,0 +1,4 @@
+<?xml version="1.0" encoding="utf-8"?>
+<svg width="32" height="32" viewBox="0 0 32 32" fill="none" xmlns="http://www.w3.org/2000/svg">
+<path d="M15.8259 4L29.6518 28H2L15.8259 4Z" fill="black"/>
```

Comment:
```suggestion
<path d="M15.8259 4L29.6518 28H2L15.8259 4Z" fill="currentColor"/>
```

Thanks for doing this! I don't have the context for this project but I think this is what we'd want to support dark/light modes out of the box?

Reply:
I checked all the other images and they all are forced to `#444444` so I've change it to that.

Wanted to check and see if the icon looks to be good for what the 32x32 size is.

![image](https://github.com/vorillaz/devicons/assets/996134/d96131e5-2101-4f8d-a71b-cbcddb773452)